src/lib/mapping.js

```diff
@@ -12,6 +13,7 @@ export default class Mapping {
     this.orMode = false;
     this.alwaysSet = options.alwaysSet;
     this.alwaysTransform = options.alwaysTransform;
+    this.defaultTransformations = [];
```

Comment:
Not sure this name is very descriptive. I think this is what we've been referring to as pipeline transformations. Maybe that's a better name? ```pipelineTransformations```?